docs/kit/adoption-view/page_adoption-view.md

```diff
@@ -13,12 +13,13 @@ The ConnectorKit provides a connector framework, based on the [Eclipse Dataspace
 
 ### Vision
 
-We believe that sharing data along the automotive supply chain adds great value for every participant. The key to such collaboration is trust in the involved partners and assurance that the shared data is not being misused. Our vision is that in 5 years, companies along the automotive value chain will share and exchange data with each other as a matter of course, without anyone fearing that their data will be misused. This creates added value for everyone, which can only be achieved through collaboration.
+We believe that sharing data along the automotive supply chain adds great value for every participant. The key to such collaboration is trust in the involved partners and assurance that the shared data is not being misused. Our vision is that in 5 years, companies along the automotive value chain will share and exchange data with each other as a matter of course, without anyone fearing that their data will be misused. This unlocks added value for everyone as well as a significantly improved collaboration.
```

Comment:
<!-- Was redundant before. Shouldn't the vision also include a vision for the "Connector Kit"? -->